README.md

```diff
@@ -78,3 +78,51 @@ be heard and the call hung up.
 To uninstall all files associated with the example issue the following command:
 
     make uninstall-example
+
+## Asterisk Channel Variables
+
+You can use the following Respoke session information to manage them inside the asterisk channel:
+
+- respoke\_session\_local
```

Comment:
your dialplan variables should probably follow asterisk convention too; which is generally `ALLUPPERCASE` without `_`